modules/materials-page/materials-page.service.js

```diff
@@ -0,0 +1,50 @@
+const Materials = require('./materials-page.model');
+const createTranslations = require('../../utils/createTranslations');
+const {
+  addTranslations,
+  updateTranslations,
+  deleteTranslations,
+} = require('../translations/translations.service');
+
+class MaterialsService {
+  async getAllMaterialsBlocks() {
+    return Materials.find({}).exec();
+  }
+
+  async addMaterialsBlock(materialsBlock) {
+    materialsBlock.translationsKey = await addTranslations(
+      createTranslations(materialsBlock)
+    );
+
+    return new Materials(materialsBlock).save();
+  }
+
+  async deleteMaterialsBlock(id) {
+    const materialsBlock = await Materials.findByIdAndDelete(id);
+
+    if (materialsBlock) {
+      await deleteTranslations(materialsBlock.translationsKey);
+
+      return materialsBlock;
+    }
+  }
+
+  async updateMaterialsBlock(id, materialsBlock) {
+    const foundMaterialsBlock = await Materials.findById(id).exec();
+
+    await updateTranslations(
+      foundMaterialsBlock.translationsKey,
```

Comment:
Here must be some check if foundMaterialBlock is undefined, because in that case here will be error

Reply:
Thank you for comments, fix)